examples/todo-list/README.md

```diff
@@ -0,0 +1,116 @@
+# @loopback/example-todo-list
+
+This is an extended tutorial from `@loopback/example-todo`.
+
+## Overview
+
+This tutorial demonstrates how to create a set of APIs for models that are
+related to another.
+
+![todo-tutorial-overview](./imgs/todo-list-overview.png)
+
+## Setup
+
+You'll need to make sure you have some things installed:
+
+- [Node.js](https://nodejs.org/en/) at v8.x or greater
+
+Additionally, this tutorial assumes that you are comfortable with certain
+technologies, languages and concepts.
+
+- JavaScript (ES6)
+- [REST](http://www.restapitutorial.com/lessons/whatisrest.html)
+
+Lastly, you'll need to install the LoopBack 4 CLI toolkit:
+
+```sh
+npm i -g @loopback/cli
+```
+
+If you're not continuing this tutorial from the
+[todo tutorial](http://loopback.io/doc/en/lb4/todo-tutorial.html), you can use
+the LoopBack 4 CLI tool to catch up to where this tutorial will continue from:
+
+```sh
+lb4 example todo
+```
+
+It should be noted that this tutorial does not assume the
+[optional geo-coding step](https://loopback.io/doc/en/lb4/todo-tutorial-geocoding-service.html)
+has been completed. Whether the step has been completed or not, the content and
+the steps listed in this tutorial remains the same.
+
+## Tutorial
+
+To follow this tutorial, begin with the
```

Comment:
instead of `begin with the ... section` it might be better to say, let's begin by [Adding a TodoList model]`.